src/swagger/objects/parameter.cr

```diff
@@ -1,3 +1,7 @@
+require "json"
+
+require "./schema"
```

Comment:
Why require json and specific object file?

Reply:
Because otherwise we can't run specific specs without requiring all the files of all the project.
I think it should be more organized, with each module requiring only what it needs to ease understanding, development / running tests and incremental refactoring.

Reply:
With all this requires added, the project organization starts to became more obvious, with which objects depends on which one;
Instead of having all files in `objects/`, subdirectories inside could be created.

Reply:
You are right.